calblueprint/meepanyar
# Tiffany/add customer frontend

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

Revised Add Customer page with new fields & styling according to Figma.

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review

- Go to http://localhost:3000/customers/create
- Add a customer

[//]: # 'The order in which to review files and what to expect when testing locally'

## Next steps

EDIT: I'm getting this error when I click on selects:

![image](https://user-images.githubusercontent.com/20329981/114948863-b6de1e80-9e04-11eb-8aa8-22cb4e4424ba.png)

I couldn't really figure out some styling details, like adjusting borderColor & adding placeholders for Select components on Material UI elements - is it okay to leave that alone for now?

Also, you can submit the form with all the required fields empty.

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

![image](https://user-images.githubusercontent.com/20329981/114818850-52be4a80-9d71-11eb-8d9e-9be3bdd57d2b.png)

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
